src/cascadia/QueryExtension/ExtensionPalette.xaml

```diff
@@ -0,0 +1,384 @@
+                            <TextBlock Margin="6,0,6,0"
+                                       VerticalAlignment="Center"
+                                       FontSize="12"
+                                       Text="{x:Bind ProfileName, Mode=OneWay}" />
+                        </StackPanel>
+                    </Border>
+                    <ListView x:Name="_suggestionsListView"
+                              Grid.Row="0"
+                              Grid.Column="0"
+                              Grid.ColumnSpan="3"
+                              HorizontalAlignment="Stretch"
+                              VerticalAlignment="Bottom"
+                              HorizontalContentAlignment="Stretch"
+                              IsItemClickEnabled="True"
+                              ItemClick="_listItemClicked"
+                              ItemTemplateSelector="{StaticResource ChatMessageTemplateSelector}"
+                              ItemsSource="{Binding Source={StaticResource MessagesCollectionViewSource}}"
+                              SelectionMode="None">
+                        <ListView.Resources>
+                            <SolidColorBrush x:Key="ListViewItemBackgroundPointerOver"
+                                             Color="Transparent" />
+                            <SolidColorBrush x:Key="ListViewItemBackgroundPressed"
+                                             Color="Transparent" />
+                        </ListView.Resources>
+                        <ListView.Header>
+                            <Grid RowSpacing="0">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                </Grid.RowDefinitions>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="1*" />
+                                    <ColumnDefinition Width="1*" />
+                                    <ColumnDefinition Width="6*" />
+                                    <ColumnDefinition Width="1*" />
+                                    <ColumnDefinition Width="1*" />
+                                </Grid.ColumnDefinitions>
+                                <mux:ImageIcon Grid.Row="0"
+                                               Grid.Column="1"
+                                               Grid.ColumnSpan="3"
+                                               Width="64"
+                                               Height="64"
+                                               Margin="0,0,0,20"
+                                               Source="ms-appx:///ProfileIcons/terminalChatLogo.png" />

```

Comment:
does this one work with light & dark theme? 

oh wait no this is the thing that has to be in dark mode isn't it